runtime/server/test_helper.go

```diff
@@ -0,0 +1,37 @@
+package server
+
+import (
+	"context"
+	"github.com/rilldata/rill/runtime/api"
+	"github.com/rilldata/rill/runtime/drivers"
+)
+
+func GetTestServer() (*Server, string, error) {
```

Comment:
nit: this should actually still be lowercase (`getTestServer`) since public/private is determined on a package-level. Actually in Go, everything is package-specific (imports as well as visibility) – files can be viewed as just a convenience to make large packages easier to navigate.

Reply:
I thought it should be capital letter so that it can be imported.

Reply:
So if this method needs to be accessed in another package then it should be capitalized ?

Reply:
anyways changed